requirements/docs.txt

```diff
@@ -6,3 +6,4 @@
 # install sphinx and its extensions
 sphinx
 sphinx-rtd-theme
+sphinx-copybutton
```

Comment:
Yeah no strong feelings either.  It's true it's not a runtime dependency and it's just one small thing that gets installed when you run `tox`. The list of things we install is long already so this doesn't change the situation much... but I don't think we should add developer dependencies without good reason -- and the copy button doesn't feel like a very important one.

The other changes seem good. I would say remove the copybutton references and we'll merge it

Reply:
That's okay @lukpueh @jku I understands your POV. BTW I've got this idea just to fulfill [last point criteria](https://github.com/cncf/techdocs/blob/main/assessments/criteria.md#new-user-content) in this heading of CNCF assessment, so that we don't miss this criteria in future. But if you like, I will remove that, and merge other changes, WDYT?
Thank you 👍

Reply:
@jku @lukpueh I've made changes, can we merge it now?
Thank you👍